.changeset/lazy-terms-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-linguist': patch
+---
+
+Get component's title from translation file
```

Comment:
Can we include a link to the docs here: https://backstage.io/docs/plugins/internationalization#for-an-application-developer-overwrite-plugin-messages. These get rolled up into the release notes so also a short explanation that this is done using internationalization would be helpful